.github/workflows/doc-index-updater-branch.yaml

```diff
@@ -1,10 +1,9 @@
 name: doc-index-updater-branch
 
 on:
-  push:
+  pull_request:
     branches:
-      - "**"
-      - "!master"
+      - "master"
```

Comment:
Might we not be interested in running tests on pull requests to any branch?

Reply:
I did think about that, but in general not so much. If branches are short-lived and getting incremental changes into master is what we're striving for, then in my view PRs into branches are suggestions to a parent master-bound PR, and so those suggestions are not worth policing and it might even hinder development.